app/controllers/stories/feeds_controller.rb

```diff
@@ -0,0 +1,38 @@
+class Stories::FeedsController < ApplicationController
+  before_action :set_cache_control_headers
```

Comment:
Since this is the version of the feed which will be dynamic and customized on a per-user basis, we don't want to set this because this is what tells Fastly to cache on the edge.

So if the logged in version of the home page is calling in for this json and we're going to generate custom collections based on our optimization of the feed algo, we'd want to ditch this header.

Reply:
Thanks for the explanation: I have a LOT to learn about how our caching works!